docker/nix/build_nix.sh

```diff
@@ -5,6 +5,13 @@ nix --version
 if [ -d "/workspace" ]; then
     cd /workspace
 fi
+SYSTEM=$(nix-instantiate --eval -E builtins.currentSystem | tr -d '"')
 nix build .#psql_15/bin -o psql_15
 nix flake check -L 
 nix copy --to s3://nix-postgres-artifacts?secret-key=nix-secret-key ./psql_15
+if [ "$SYSTEM" = "aarch64-linux" ]; then
+    nix build .#postgresql_15_debug -o ./postgresql_15_debug
+    nix build .#postgresql_15_src -o ./postgresql_15_src
+    nix copy --to s3://nix-postgres-artifacts?secret-key=nix-secret-key ./postgresql_15_debug-debug
```

Comment:
I could see how this could be correct, but just to double check: There isn't an extra debug suffix here is there?

Reply:
nix unfortunately introduces that extra suffix, so it is correct, although not great.

Reply:
I'll follow up soon to see if I can correct it as the source, but for now I just want to make sure this is available to our team.